x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/models_management/models_list.tsx

```diff
@@ -278,12 +280,19 @@ export const ModelsList: FC = () => {
       type: 'icon',
       available: (item) => item.metadata?.analytics_config?.id,
       onClick: async (item) => {
-        await navigateToUrl(
-          getResultsUrl(
-            item.metadata?.analytics_config.id,
-            Object.keys(item.metadata?.analytics_config.analysis)[0]
-          )
-        );
+        if (item.metadata?.analytics_config === undefined) return;
+
+        const url = await urlGenerator.createUrl({
+          page: ML_PAGES.DATA_FRAME_ANALYTICS_EXPLORATION,
+          pageState: {
+            jobId: item.metadata?.analytics_config.id as string,
+            analysisType: Object.keys(
+              item.metadata?.analytics_config.analysis
+            )[0] as DataFrameAnalyticsType,
```

Comment:
We have `getAnalysisType()` available in `plugins/ml/public/application/data_frame_analytics/common/analytics.ts`.

Reply:
Updated here https://github.com/elastic/kibana/pull/76735/commits/cc8f5dd3c6567646ceea67439dd8544e8fb7789d